packages/mui-base/src/SelectUnstyled/SelectUnstyled.tsx

```diff
@@ -288,6 +318,13 @@ SelectUnstyled.propTypes /* remove-proptypes */ = {
    * Use in controlled mode (see listboxOpen).
    */
   onListboxOpenChange: PropTypes.func,
+  /**
+   * A function used to convert the option value to a string.
+   * It is used to navigate through values with keyboard.
+   *
+   * @default defaultOptionStringifier
+   */
+  optionStringifier: PropTypes.func,
```

Comment:
```suggestion
  getStringifiedOption: PropTypes.func,
```
Maybe? I am not sure, but I would use `getX` as a prefix.

Reply:
I'm fine with both. It's an existing prop, however, so I wouldn't rename it in this PR.